sdk/spring/azure-spring-boot-samples/azure-spring-cloud-sample-eventhubs-binder/README.md

```diff
@@ -189,18 +191,42 @@ spring:
       region: [region]
 ```
 
+#### Enable sync message
+If you want to get a response from eventhub immediately after sending a message, 
```

Comment:
How about writing like: 
To enable message sending in a synchronized way with Spring Cloud Stream 3.x, `azure-spring-cloud-stream-binder-eventhubs` supports the sync producer mode to get responses for sent messages. By enabling following configuration, you could use [StreamBridge](https://docs.spring.io/spring-cloud-stream/docs/3.1.3/reference/html/spring-cloud-stream.html#_sending_arbitrary_data_to_an_output_e_g_foreign_event_driven_sources) for the synchronized message producing.